digitalpy/core/zmanager/action_mapper.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D100 Missing docstring in public module